packages/0x.js/package.json

```diff
@@ -100,11 +101,11 @@
         "@0xproject/assert": "^0.2.9",
         "@0xproject/base-contract": "^0.3.1",
         "@0xproject/contract-wrappers": "^0.0.1",
-        "@0xproject/order-watcher": "^0.0.1",
         "@0xproject/order-utils": "^0.0.4",
+        "@0xproject/order-watcher": "^0.0.1",
+        "@0xproject/sol-compiler": "^0.4.3",
         "@0xproject/types": "^0.6.3",
         "@0xproject/typescript-typings": "^0.3.1",
-        "@0xproject/sol-compiler": "^0.4.3",
```

Comment:
It looks like you are running prettier on this file... We added `package.json` to `.prettierignore`, is it not adhering to that in your IDE?

Reply:
@LogvinovLeon why did we add it to gitignore? It might be better to use prettier here.

Reply:
I didn't edit any __package.json__ files directly. It was all done via `lerna add`.